solr/core/src/java/org/apache/solr/cluster/placement/impl/NodeMetricImpl.java

```diff
@@ -67,7 +67,7 @@ public NodeMetricImpl(String key) {
 
   public NodeMetricImpl(String key, Function<Object, T> converter) {
     super(key, key, converter);
```

Comment:
Maybe replace implementation by `this(key, Registry.UNSPECIFIED, key, converter);` ?